test/integration/get-community-controller.spec.ts

```diff
@@ -14,6 +14,13 @@ test('Get Community Controller', function ({ components, spyComponents }) {
     beforeEach(async () => {
       identity = await createTestIdentity()
       address = identity.realAccount.address.toLowerCase()
+      
+      // Mock the comms gatekeeper to return a default voice chat status
+      spyComponents.commsGatekeeper.getCommunityVoiceChatStatus.mockResolvedValue({
```

Comment:
Shall we create a new context (describe) to state this? such like: `when there is an active voice chat room`

Reply:
the thing is, all base tests will need it and we assert that the data is there here: https://github.com/decentraland/social-service-ea/pull/196/files#diff-2826030a371cada984b2af452b168ad76dd534185f274aac4b7fa0b0fc934850R72.
I added the `and the community has no active voice chat'` to test the opposite thing. 
Do you think we still need to split them? it might get too messy to read the tests, I thought this was cleaner